Feature/package updates (#7)
fivetran/dbt_youtube_analytics_source: README.md

<p align="center">
    <a alt="License"

>         href="https://github.com/fivetran/dbt_jira_source/blob/main/LICENSE">

Reviewer: This is pointing to jira. I imagine we want this to be the suggested.
```suggestion
        href="https://github.com/fivetran/dbt_youtube_analytics_source/blob/main/LICENSE">
```

        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
    <a alt="Fivetran-Release"
        href="https://fivetran.com/docs/getting-started/core-concepts#releasephases">
        <img src="https://img.shields.io/badge/Fivetran Release Phase-_Beta-orange.svg" /></a>
    <a alt="dbt-core">
        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.0.0_<2.0.0-orange.svg" /></a>
    <a alt="Maintained?">
        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
    <a alt="PRs">
        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
</p>

> # YouTube Source dbt Package ([Docs](https://fivetran.github.io/dbt_youtube_source/))

Reviewer: ```suggestion
# YouTube Source dbt Package ([Docs](https://fivetran.github.io/dbt_youtube_analytics_source/))
```

> # YouTube Source dbt Package ([Docs](https://fivetran.github.io/dbt_youtube_source/))

Reviewer: ```suggestion
# YouTube Analytics Source dbt Package ([Docs](https://fivetran.github.io/dbt_youtube_source/))
```

# 📣 What does this dbt package do?

> - Cleans, tests, and prepares your Youtube data from [Fivetran's connector](https://fivetran.com/docs/applications/youtube) for analysis.

Reviewer: ```suggestion
- Cleans, tests, and prepares your Youtube data from [Fivetran's connector](https://fivetran.com/docs/applications/youtube-analytics) for analysis.
```

> - Generates a comprehensive data dictionary of your Youtube data through the [dbt docs site](https://fivetran.github.io/dbt_youtube_source/)

Reviewer: ```suggestion
- Generates a comprehensive data dictionary of your Youtube data through the [dbt docs site](https://fivetran.github.io/dbt_youtube_analytics_source/)
```

> - Materializes staging tables that leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/youtube/#schemainformation). These tables are designed to work simultaneously with our [Youtube modeling package](https://github.com/fivetran/dbt_youtube). Learn more about these models on the [dbt docs site](https://fivetran.github.io/dbt_youtube_source/#!/overview/jira_source/models/?g_v=1).

Reviewer: ```suggestion
- Materializes staging tables that leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/youtube-analytics/#schemainformation). These tables are designed to work simultaneously with our [Youtube modeling package](https://github.com/fivetran/dbt_youtube_analytics). Learn more about these models on the [dbt docs site](https://fivetran.github.io/dbt_youtube_analytics_source/#!/overview?g_v=1&g_e=seeds).
```

# 🎯 How do I use the dbt package?
## Step 1: Prerequisites
To use this dbt package, you must have the following:

> - At least one Fivetran Youtube connector syncing data into your destination. 

Reviewer: Just to be consistent
```suggestion
- At least one Fivetran Youtube Analytics connector syncing data into your destination. 
```

- A **BigQuery**, **Snowflake**, **Redshift**, **Databricks**, or **Postgres** destination.

## Step 2: Install the package
Include the following youtube_source package version in your `packages.yml` file.
> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
```yaml
packages:
  - package: fivetran/youtube_analytics_source
    version: [">=0.3.0", "<0.4.0"]
```
## Step 3: Define database and schema variables

> By default, this package runs using your destination and the `youtube` schema. If this is not where your Youtube data is (for example, if your Youtube schema is named `youtube_fivetran`), add the following configuration to your root `dbt_project.yml` file:

Reviewer: ```suggestion
By default, this package runs using your destination and the `youtube_analytics` schema. If this is not where your Youtube data is (for example, if your Youtube schema is named `youtube_fivetran`), add the following configuration to your root `dbt_project.yml` file:
```

```yml
vars:
    youtube_database: your_destination_name
    youtube_schema: your_schema_name 
```

> ## Step 4: Required configurations

Reviewer: I feel like this step is more important than Step 3. Can we switch Step 3 and Step 4 around.

### Disable Demographics Report
This packages assumes you are syncing the YouTube `channel_demographics_a1` report. If you are not syncing this report, you may add the below configuration to your `dbt_project.yml` to disable the `stg_youtube__demographics` model and all downstream references.
```yml
# dbt_project.yml

...
vars:
  youtube__using_channel_demographics: false # true by default
```
## Required YouTube Channel Reports
To use this package you will need to pull the following YouTube Analytics reports through Fivetran:
- [channel_basic_a2](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-user-activity) (required)
- [channel_demographics_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-viewer-demographics) (optional)
- [videos metadata table](https://resources.fivetran.com/datasheets/youtube-metadata-cloud-function-guide-2) (optional)

## (Optional) Step 5: Additional configurations
<details><summary>Expand to view configurations</summary>

> ### YouTube Video Metadata

Reviewer: Cab we have this follow the pattern for the other headers. I am think of something like 
```
### Including YouTube Video Metada
```

```

</details>

## (Optional) Step 6: Orchestrate your models with Fivetran Transformations for dbt Core™
<details><summary>Expand to view details</summary>
<br>

Fivetran offers the ability for you to orchestrate your dbt project through [Fivetran Transformations for dbt Core™](https://fivetran.com/docs/transformations/dbt). Learn how to set up your project for orchestration through Fivetran in our [Transformations for dbt Core setup guides](https://fivetran.com/docs/transformations/dbt#setupguide).
</details>

# 🔍 Does this package have dependencies?
This dbt package is dependent on the following dbt packages. For more information on the following packages, refer to the [dbt hub](https://hub.getdbt.com/) site.
> IMPORTANT: If you have any of these dependent packages in your own `packages.yml` file, we highly recommend that you remove them to avoid package version conflicts.
```yml
packages:
    - package: fivetran/fivetran_utils
      version: [">=0.3.0", "<0.4.0"]
    - package: dbt-labs/dbt_utils
      version: [">=0.8.0", "<0.9.0"]
```
          
# 🙌 How is this package maintained and can I contribute?
## Package Maintenance

> The Fivetran team maintaining this package _only_ maintains the latest version of the package. We highly recommend that you stay consistent with the [latest version](https://hub.getdbt.com/fivetran/jira_source/latest/) of the package and refer to the [CHANGELOG](https://github.com/fivetran/dbt_jira_source/blob/main/CHANGELOG.md) and release notes for more information on changes across versions.

Reviewer: ```suggestion
The Fivetran team maintaining this package _only_ maintains the latest version of the package. We highly recommend that you stay consistent with the [latest version](https://hub.getdbt.com/fivetran/youtube_analytics_source/latest/) of the package and refer to the [CHANGELOG](https://github.com/fivetran/dbt_youtube_analytics_source/blob/main/CHANGELOG.md) and release notes for more information on changes across versions.
```

## Contributions
A small team of analytics engineers at Fivetran develops these dbt packages. However, the packages are made better by community contributions! 

We highly encourage and welcome contributions to this package. Check out [this dbt Discourse article](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) to learn how to contribute to a dbt package!

# 🏪 Are there any resources available?

> - If you have questions or want to reach out for help, please refer to the [GitHub Issue](https://github.com/fivetran/dbt_jira_source/issues/new/choose) section to find the right avenue of support for you.

Reviewer: ```suggestion
- If you have questions or want to reach out for help, please refer to the [GitHub Issue](https://github.com/fivetran/dbt_youtube_analytics_source/issues/new/choose) section to find the right avenue of support for you.
```
